addons/common/XEH_postInit.sqf

```diff
@@ -96,7 +96,7 @@ if !(isDedicated) then {
                 [GVAR(actionList), {
                     TRACE_3("Inside the code for the hashPair",(vehicle player),GVAR(actionIndexes), _value);
                     if (!isNil "_value" && typeName(_value) == "ARRAY") then {
-                        PUSH(GVAR(actionIndexes), (vehicle player) addAction _value)
+                        GVAR(actionIndexes) pushBack (vehicle player) addAction _value

```

Comment:
Yeah. This is missing the brackets too.
Evaluations of binary commands are done from left to right:
`GVAR(actionIndexes) pushBack (vehicle player) addAction _value`
->
`<index/number> addAction _value`
->
`Error Number exected object`

The brackets around `(vehicle player)` on the other hand are optional.

Correct and minimalistic:
`GVAR(actionIndexes) pushBack (vehicle player addAction _value)

I personally think that it's best to use as few brackets as possbile. Only exception would be
`(unaryCommand _var1) binaryCommand _var2`, so in the end exactly what killswitch proposed.